### PR TITLE
fix: stealth address package imports

### PR DESCRIPTION
**Pull request by marcomariscal**

### CODE REVIEW

File: examples/checkStealthAddress/index.ts [hunk 1]
```diff
@@ -24,12 +24,12 @@ console.log(`View Tag: ${viewTag}`);
...
-  ephemeralPublicKey, // From the announcement
-  schemeId: VALID_SCHEME_ID.SCHEME_ID_1,
-  spendingPublicKey: userSpendingPublicKey,
-  userStealthAddress: stealthAddress, // User's known stealth address
-  viewingPrivateKey: userViewingPrivateKey,
-  viewTag, // From the announcement
+	ephemeralPublicKey, // From the announcement
+	schemeId: VALID_SCHEME_ID.SCHEME_ID_1,
+	spendingPublicKey: userSpendingPublicKey,
+	userStealthAddress: stealthAddress, // User's known stealth address
```
Comment (garyghayrat) [inline examples/checkStealthAddress/index.ts:30]: As discussed let's remove the formatting changes across the PR if they were not intended to be there.

Comment (marcomariscal) [reply examples/checkStealthAddress/index.ts]: ran `bun run format` here; must have unintentionally been using biome or some stale format method.

https://github.com/ScopeLift/stealth-address-sdk/pull/64/commits/d5b2ae4243fdc992732a820c0accf509600f3d2d

Comment (marcomariscal) [reply examples/checkStealthAddress/index.ts]: should be minimal formatting diffs now

**Review by garyghayrat — APPROVED** [review]

**Comment by marcomariscal** [comment]
closing this in favor of (couldn't figure out why rebasing was pulling in a massive main commit):

https://github.com/ScopeLift/stealth-address-sdk/pull/65

**Pull request closed without merging**